Feature accessibility

IsmailChangezi opened the pull request:
- Structured the page with html
- styled the page with css
- added functionality with JS
- Used webpack to parcel JS
- Fixed Linter errors
-

AmaduKamara reviewed (changes requested):
# Status: Required Changes ♻️♻️♻️ 

Hi @IsmailChangezi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ App works fine as expected 👍 

## Required Changes ♻️
_Check the comments under the review._

- [ ] You're using the required README Template recommended by Microverse, but it's empty at the moment. You have to fill most of the areas in it with descriptions of the current app you are working on instead of leaving it blank as it's. The template is meant for us to use and add content about separate projects we work on and about the author as well. Please fill them up 👍

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

mahbubul14 commented on README.md:
```diff
@@ -1,45 +1,272 @@
+> Describe your project in 1 or 2 sentences.
+
+**[your_project__name]** is a...
+
```
- [ ] Kindly add your project name and describe your project in a few sentences. :+1:

mahbubul14 commented on README.md:
```diff
@@ -1,45 +1,272 @@
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
- [ ] Kindly remove this part. The link is not working. :+1:

mahbubul14 commented on README.md:
````diff
@@ -1,45 +1,272 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
 ## Getting Started
 
 1. In the console run npm install.
 2. Then run npm start.
 
-## Authors
+### Prerequisites
+
+In order to run this project you need:
+ 
+ The basic understanding of HTML,CSS, JS
+ 
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
````
- [ ] Kindly update the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1:

mahbubul14 reviewed (changes requested):
Hi @IsmailChangezi  ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

1GORDON reviewed (changes requested):
##
## Hi @IsmailChangezi ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] I can see you have deleted the readme file yet it is a mandatory requirement for all Microverse projects. Kindly add the correct readme file and customize it accordingly as suggested by the previous reviewer. You may use this [readme template](https://github.com/microverseinc/readme-template) to get started.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lameck1 reviewed (commented):
Hello @IsmailChangezi 👋,

_This is an additional review due to the second opinion request._

During code reviews, the code reviewer is only interested in the files submitted in the PR. Thus, the branch submitting this PR must have the README file and other files containing the code related to the project being submitted for review. Deleting the README file was not the solution here, you could have addressed the concerns and your project could have been approved by now.

Besides this, I would like to apologize on behalf of @AmaduKamara who suggested changes to the README. @AmaduKamara, when the README is not professionally documented as expected, we should not review the PR but INVALIDATE so as not to consume the student's review limits for nothing. @1GORDON, and @mahbubul14, please take note of this too.

@IsmailChangezi, you have one more chance to make things right. Please Add back the README file to this branch and then document it properly. When done, please drop a reply here. We'll check your work again and approve your project when we realize all the concerns raised on your README file have been addressed.

Keep rocking 🚀!

IsmailChangezi commented:
Hi Microverse,
I have completed the required task.
You can go on and check the PR.
Thank you.

On Fri, 9 Dec 2022 at 17:20, Lameck Otieno ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> Hello @IsmailChangezi <https://github.com/IsmailChangezi> 👋,
>
> *This is an additional review due to the second opinion request.*
>
> During code reviews, the code reviewer is only interested in the files
> submitted in the PR. Thus, the branch submitting this PR must have the
> README file and other files containing the code related to the project
> being submitted for review. Deleting the README file was not the solution
> here, you could have addressed the concerns and your project could have
> been approved by now.
>
> Besides this, I would like to apologize on behalf of @AmaduKamara
> <https://github.com/AmaduKamara> who suggested changes to the README.
> @AmaduKamara <https://github.com/AmaduKamara>, when the README is not
> professionally documented as expected, we should not review the PR but
> INVALIDATE so as not to consume the student's review limits for nothing.
> @1GORDON <https://github.com/1GORDON>, and @mahbubul14
> <https://github.com/mahbubul14>, please take note of this too.
>
> @IsmailChangezi <https://github.com/IsmailChangezi>, you have one more
> chance to make things right. Please Add back the README file to this branch
> and then document it properly. When done, please drop a reply here. We'll
> check your work again and approve your project when we realize all the
> concerns raised on your README file have been addressed.
>
> Keep rocking 🚀!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/IsmailChangezi/to-do-list-2/pull/2#pullrequestreview-1211584606>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW5MGIXPO7DRP65BBEZGOGDWMMPZRANCNFSM6AAAAAASYN6C2Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Lameck1 reviewed (approved):
Hello, @IsmailChangezi,

This is a manual review!

Well done on addressing the concerns pointed out in the previous reviews.

As the README file was the only concern that made this PR not get approved, your PR will be approved ASAP in the next few minutes. Please be ready to submit this task in your dashboard.

Cheers, keep rocking 🚀.

IsmailChangezi commented:
Thank you so much.

On Fri, 9 Dec 2022 at 6:21 pm, Lameck Otieno ***@***.***>
wrote:

> ***@***.**** approved this pull request.
>
> Hello, @IsmailChangezi <https://github.com/IsmailChangezi>,
>
> This is a manual review!
>
> Well done on addressing the concerns pointed out in the previous reviews.
>
> As the README file was the only concern that made this PR not get
> approved, your PR will be approved ASAP in the next few minutes. Please be
> ready to submit this task in your dashboard.
>
> Cheers, keep rocking 🚀.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/IsmailChangezi/to-do-list-2/pull/2#pullrequestreview-1211658153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW5MGIWCIFFK2OX7DUNWQT3WMMW7FANCNFSM6AAAAAASYN6C2Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
